src/components/MenuPanel/MenuPanel.js

```diff
@@ -232,26 +240,29 @@ class AnimatedMenuPanel extends React.Component {
             </Wrap>
           )}
         </Spring>
-        <Overlay opened={opened} onClick={onCloseMenuPanel} />
+        <Viewport>
+          {({ below }) =>
+            below('medium') && (
+              <Overlay opened={!!openProgress} onClick={onCloseMenuPanel} />
+            )
+          }
+        </Viewport>
       </React.Fragment>
     )
   }
 }
 
+AnimatedMenuPanel.propTypes = {
+  openProgress: PropTypes.number.isRequired,
+  autoClosing: PropTypes.bool.isRequired,
+  onCloseMenuPanel: PropTypes.func.isRequired,
+}
+
 const Overlay = styled.div`
   position: absolute;
   z-index: 2;
-  width: 100vw;
+  width: ${({ opened }) => (opened ? '100vw' : '1px')};
```

Comment:
Could we leave a few comments explaining why we need this overlay (e.g. required for edge swipe on Android)?